l10n_ch_pain_credit_transfer/tests/test_ch_sct.py

```diff
@@ -6,331 +6,318 @@
 
 from lxml import etree
 
+from odoo.tests.common import SavepointCase
 from odoo.tools import float_compare
 
-from odoo.addons.account.tests.account_test_classes import AccountingTestCase
-
 ch_iban = "CH15 3881 5158 3845 3843 7"
 
 
-class TestSCTCH(AccountingTestCase):
-    def setUp(self):
-        super().setUp()
-        Account = self.env["account.account"]
-        Journal = self.env["account.journal"]
-        PaymentMode = self.env["account.payment.mode"]
+class TestSCTCH(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
 
-        self.payment_order_model = self.env["account.payment.order"]
-        self.payment_line_model = self.env["account.payment.line"]
-        self.bank_line_model = self.env["bank.payment.line"]
-        self.partner_bank_model = self.env["res.partner.bank"]
-        self.attachment_model = self.env["ir.attachment"]
-        self.invoice_model = self.env["account.invoice"]
-        self.invoice_line_model = self.env["account.invoice.line"]
+        cls.main_company = cls.env.ref("base.main_company")
+        cls.partner_deco = cls.env.ref("base.res_partner_2")
```

Comment:
Pleas avoid mentioning the partner's name in the variable, otherwise next time it changes we will have again a big diff for nothing.